stress-test/pipeline.yaml

```diff
@@ -0,0 +1,144 @@
+##############################################################################
+# YAML Anchors
+##############################################################################
+infrastructure_vars: &infrastructure_vars
+  TF_VAR_gcp_folder_id: ((project.folder_id))
+  TF_VAR_gcp_billing_account: ((project.billing_account))
+  TF_VAR_project_env: ((project.label))
+  TF_VAR_machine_type: ((k8s.node_machine_type))
+  TF_VAR_project_id: ((project.id))
+
+benchmark_vars: &benchmark_vars
+  PROJECT_ID: ((project.id))
+  REGION: ((project.region))
+  REQUESTS_JSON: ((benchmark.requests_json))
+  LOCUST_OPTIONS: '--no-web --clients ((benchmark.clients_per_instance)) --hatch-rate ((benchmark.hatch_rate)) --run-time ((benchmark.run_time)) --csv=output -L WARNING'
+  RUNNER_FULLY_QUALIFIED_DOMAIN_NAME: ((benchmark.target_fully_qualified_domain_name))
+  DOCKER_REGISTRY: ((docker.app_registry))
+  IMAGE_TAG: ((benchmark.image_tag))
+  USER_WAIT_TIME_MIN_SECONDS: ((benchmark.user_wait_time_min_seconds))
+  USER_WAIT_TIME_MAX_SECONDS: ((benchmark.user_wait_time_max_seconds))
+  REQUESTED_CPU_PER_POD: ((k8s.available_cpu_per_pod))
+  REQUESTED_MEMORY_PER_POD: ((k8s.available_memory_per_pod))
+  PARALLELISM: ((benchmark.instances))
+  TIMEOUT: ((k8s.timeout))
+
+##############################################################################
+# Resources
+##############################################################################
+
+resource_types:
+  - name: slack-notification
+    type: docker-image
+    source:
+      repository: arbourd/concourse-slack-alert-resource
+      tag: latest
+
+resources:
+
+##############################################################################
+# Git Repositories
+##############################################################################
+
+  - name: eq-survey-runner-benchmark
+    type: git
+    source:
+      uri: https://github.com/ONSdigital/eq-survey-runner-benchmark.git
+      branch: stress-test
+
+  - name: eq-terraform-load-generator
+    type: git
+    source:
+      uri: https://github.com/ONSdigital/eq-terraform-load-generator.git
+      branch: stress-test
+
+##############################################################################
+# Misc Resources
+##############################################################################
+
+  - name: slack-alert
+    type: slack-notification
+    source:
+      url: ((slack.webhook))
+      channel: '#((slack_channel.name))'
+
+jobs:
+
+##############################################################################
+# Deploy Infrastructure
+##############################################################################
+
+  - name: deploy-infrastructure
```

Comment:
What about the job to publish the benchmark image to GCR?

Reply:
Not needed - we are pulling the benchmark image using `latest` or branch tags

Reply:
The benchmark is already published to GCR https://github.com/ONSdigital/eq-survey-runner-benchmark/blob/master/.travis.yml - pulling from there with a tag works and the job isn't needed